specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2132,6 +2132,16 @@
         }
       }
     },
+    "EncryptionIdentity":{
+       "description": "Encryption Identity properties when storage account is associated with a UserAssignedIdentity.",
+       "properties": {
+         "userAssignedIdentity":{
+           "type": "string",
+           "description": "ResourceID of the UserAssignedIdentity to be associated with the storage account.",
```

Comment:
"ResourceID of the UserAssignedIdentity to be associated with server-side encryption on the storage account."